scripts/download-aquifer-pdfs.py

```diff
@@ -0,0 +1,26 @@
+"""
```

Comment:
This is the scrape script use to get documents from ESRI's cloud based storage. Should it be stored with the main codebase?

Reply:
Yes please. Just have to confirm with @DerekRoberts where it should live. Also, please add license info to the top.

Reply:
@DerekRoberts where should I put the script that's used to import attachments from ESRI's cloud?

Reply:
./app/scripts please!